Documentation/ApiOverview/Icon/Index.rst

```diff
@@ -79,3 +79,35 @@ You can also simple use the Fluid ViewHelper to render an icon in your view:
    {namespace core = TYPO3\CMS\Core\ViewHelpers}
    <core:icon identifier="my-icon-identifier" size="small" overlay="overlay-identifier" />
 
+.. _available-icons:
+
+Available Icons
+===============
+
+The TYPO3 Core comes with a number of icons that may be used in your extensions.
+
+To search for available icons, you can use one of these possibilities:
+
+Install Extension Styleguide
```

Comment:
This should be `Install Styleguide Extension`, otherwise it should be `Install Extension Styleguide Extension`. ;-)